src/utils/window.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Defines a subset of FullStory browser APIs used by DLO.
+ */
+export interface FullStoryAPI {
```

Comment:
Is this vestigial now that we have the FS mock and the logger uses the listener pattern?

Reply:
Good catch; yes.